people/ondrej-bojar.md

```diff
@@ -0,0 +1,12 @@
+---
+grand_parent: Community
+parent: People
+title: Ondřej Bojar
+description: Machine translation researcher
+---
+
+**Ondřej Bojar** is a researcher and associate professor at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague, widely cited for his work on statistical and later [neural machine translation](/neural-machine-translation).
+He co-authored Moses, a kit for [statistical machine translation](/statistical-machine-translation) and is a long-term organizer of [WMT](/community/communities.md#workshop_on_statistical_machine_translation).
+
+His work goes beyond mainstream machine translation, such as interactive machine translation, post-editing or psycholinguistical aspects of machine translation usage.
+He managed Charles University's participation in the [Bergamot](https://browser.mt/) project for browser-based machine translation and the [ELITR](https://elitr.eu/) project for simultaneous translation of European languages and is the author of [CzEng](https://ufal.mff.cuni.cz/czeng), the Czech-English parallel corpus with over 60 million authentic sentence pairs.
```

Comment:
I would also separate this sentence into two for better performance when we machine translate the site. 😉

Reply:
It doesn't look nice stylistically to have so many sentences start with "He ..." but I see your point.